src/Utility.ts

```diff
@@ -0,0 +1,58 @@
+import * as core from '@actions/core';
+import * as exec from '@actions/exec';
+import util = require('util');
+const cpExec = util.promisify(require('child_process').exec);
```

Comment:
Same comment here about using `@actions/exec` instead

Reply:
`az` commands did not execute with `@actions/exec` for some reason. Referred to [Azure/cli](https://github.com/Azure/cli/blob/master/src/main.ts#L28) and found `az` commands to execute correctly if they are executed using `util.promisify(require('child_process').exec)` instead.